web/libs/datamanager/src/components/Common/Range/Range.scss

```
@@ -0,0 +1,132 @@
+.range {
+  display: inline-flex;
+  align-items: center;
+  user-select: none;
+
+  &_align {
+    &_horizontal {
+      min-width: 120px;
+      height: 28px;
+      flex-direction: row;
+    }
+
+    &_vertical {
+      min-height: 120px;
+      width: 28px;
+      flex-direction: column;
+    }
+
+    .range_with-icon &_horizontal {
+      padding: 0 5px;
+    }
+
+    .range_with-icon &_vertical {
+      padding: 5px 0;
+    }
+  }
+
+  &__body {
+    position: relative;
+
+    .range_align_horizontal & {
+      height: 4px;
+      width: 100%;
+    }
+
+    .range_align_vertical & {
+      height: 100%;
+      width: 4px;
+    }
+
+    .range_align_horizontal &_with-icon {
+      margin: 0 5px;
+    }
+
+    .range_align_vertical &_with-icon {
+      margin: 5px 0;
+    }
+  }
+
+  &__line {
+    background: linear-gradient(0deg, rgba(0, 0, 0, 0.05), rgba(0, 0, 0, 0.05)), #FFFFFF;
```

Flagging as opportunity for css var replacement. Further usage review needed.
```suggestion
    background: linear-gradient(0deg, rgba(var(--black), 0.05), rgba(var(--black), 0.05)), #FFFFFF;
```